### DISCUSSION TITLE
When rendering Katex mathematical formulas, a scroll bar always appears below.

### DISCUSSION THREAD

**Discussion by timeti123553** [squidfunk/mkdocs-material#7033] [Q&A]
Hello!
When I use KaTeX to write mathematical formulas, such as the following formula, after rendering, a slider always appears below the formula:

```
$$
\begin{align}
  \left\langle\boldsymbol{P}^{2}\right\rangle&=(\Delta p)^{2} \gtrsim\frac{\hbar^{2}}{ (\Delta r)^{2}} \\
  E&=\langle T+V\rangle \gtrsim \frac{\hbar^{2}}{2 \mu(\Delta r)^{2}}-\frac{\alpha}{r_{0}^{s}}
\end{align}
$$
```

![output](https://github.com/squidfunk/mkdocs-material/assets/159710240/27fac38f-afb9-427a-8ab0-c71b58b56fbd)

After I modified the page width, the scroll bar below the formula still cannot be eliminated. Because this scroll bar is redundant, the mathematical formula doesn't seem to require much of a scroll bar.

![output](https://github.com/squidfunk/mkdocs-material/assets/159710240/3f317af8-dfb5-4b9f-91be-f656b2b16fba)

Does anyone know how to eliminate the scroll bar below?

**Comment by HaitongLan** [comment]
Hello,I had the same problem. Add the following code to the.css file can solve this problem
`  .katex-display > .katex {
    max-width: 100% !important;
    overflow-x: hidden !important;
    white-space: normal !important;
}`
